lucida/template/cpp/server/Parser.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
```

Comment:
That's for the old version template for reading the port config file. Right now we are thinking of using dynamic way (databases) to store the port info. So this may be deleted